includes/class-wc-stripe-webhook-handler.php

```
@@ -1759,4 +1781,137 @@ private function get_payment_method_type_from_charge( $charge ) {
 	}
+
+	/**
+	 * Processes an agentic checkout session after the concurrency lock is acquired.
+	 *
+	 * @since 10.6.0
+	 * @param object $notification The webhook notification from Stripe.
+	 */
+	private function handle_agentic_checkout_session( $notification ): void {
+		$checkout_session = $notification->data->object;
+
+		if ( ! WC_Stripe_Feature_Flags::is_agentic_commerce_enabled() ) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Feature_Flags' in method 'handle_agentic_checkout_session'. (undefined)

(StaticAccess)